packages/components/src/internal/components/editable/models.ts

```diff
@@ -287,6 +287,19 @@ export class EditorModel
         return values.asImmutable();
     }
 
+    getColumnFromMap(fieldKeyOrName: string): QueryColumn {
+        // first try to find by fieldKey (lowercase)
+        let col = this.columnMap.get(fieldKeyOrName.toLowerCase());
+
+        // second try to find by fieldKey or name (case-sensitive): Issue 52132
```

Comment:
Do we know we need to be case-sensitive here?

Reply:
No. The columnMap uses lowerCase keys so I was thinking this would be a way to double check the field key if by some chance we didn't put something into columnMap using lower. But I agree that we should just toLowerCase the fieldKey and names in this check as well.